src/components/Pagination.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Pagination extends Component{
+  render(){
+    return(
+      <div>
+        <ul className="pagination">
+          <li className="page-item disabled">
+            <button className="page-link" >&laquo;</button>
+          </li>
+          <li className="page-item active">
+            <button className="page-link" onClick = {() => this.props.handleClick(1)}>1</button>
+          </li>
+          <li className="page-item">
+            <button className="page-link" onClick = {() => this.props.handleClick(2)}>2</button>
+          </li>
+          <li className="page-item">
+            <button className="page-link"  onClick = {() => this.props.handleClick(3)}>3</button>
+          </li>
+          <li className="page-item">
+            <button className="page-link" onClick = {() => this.props.handleClick(4)}>4</button>
+          </li>
+          <li className="page-item">
+            <button className="page-link" onClick = {() => this.props.handleClick(5)}>5</button>
+          </li>
+          <li className="page-item">
+            <button className="page-link" >&raquo;</button>
+          </li>
+        </ul>
+      </div>
+    )
+  }
+}
+
+Pagination.PropTypes={
```

Comment:
Currently you can omit required props and there will be no warnings.
In order for the check to work, use
```
Pagination.propTypes = {
  handleClick: PropTypes.func.isRequired
}
``` here and in other places.